docs/install/install_everest_helm_charts.md

```diff
@@ -38,15 +38,72 @@ Here are the steps to install Percona Everest and deploy additional database nam
 
             You can override the name of the database namespace by using the `dbNamespace.namespaceOverride` parameter. If you prefer to deploy just the core components, set `dbNamespace.enabled=false`
 
-    !!! note
-        PMM can now be deployed as a sub-chart by setting `pmm.enabled=true`. PMM will be automatically deployed within the `everest-system` namespace.
-
-        **Example**
+    
+    - (**Optional**) PMM can be deployed as a sub-chart by setting `pmm.enabled=true`. PMM will be automatically deployed within the `everest-system` namespace.
 
         ```sh
         helm install everest-core percona/everest --namespace=everest-system --create-namespace --set pmm.enabled=true
         ```
 
+    ??? info " 🌐 Install Percona Everest and access it using Ingress"
+
+        **Prerequisite**
+
+        - Ensure that you have an Ingress controller (e.g., NGINX) installed and properly configured in your cluster.
+
+        - When using TLS, `everest-tls` secret should contain a valid TLS certificate for your domain.
+
+        **Example**
+
+        To install Percona Everest and access using [Ingress](https://kubernetes.io/docs/concepts/services-networking/ingress/#what-is-ingress){:target="_blank"}, here are the steps:
+        {.power-number}
+
+        1. Update your Helm chart configuration:
+
+            ```sh
+            ingress:
+            # -- Enable ingress for Everest server
+            enabled: false
+            # -- Ingress class name. This is used to specify which ingress controller should handle this ingress.
+            ingressClassName: "nginx"
+            # -- Additional annotations for the ingress resource.
+            annotations: {}
+            # -- List of hosts and their paths for the ingress resource.
+            hosts:
+                - host: everest.example.com
+                  paths:
+                    - path: /
+                      pathType: ImplementationSpecific
+            # -- TLS configuration for the ingress resource.
+            # -- Each entry in the list specifies a TLS certificate and the hosts it applies to.
+            tls: []
+            #  - secretName: everest-tls
+            #    hosts:
+            #      - everest.example.com
+            ```
+
+        2. Install Percona Everest:
+
+            ```sh
+            helm --install everest percona/everest \
+            -n everest-system \
+            --set ingress.enabled=true \
+            --set ingress.ingressClassName="" \
+            --set ingress.hosts[0].host=everest.example.com \
+            --set ingress.hosts[0].paths[0].path=/ \
+            --set  ingress.hosts[0].paths[0].pathType=ImplementationSpecific
+            ```
+            Replace `everest.example.com` with your own domain.
```

Comment:
Not sure why we have 2 separate steps to achieve the same thing? In step 1, we're configuring a YAML values file, but in step 2 we are not using it -- instead we're setting the same using the `--set` command. We should either set the params using `--set` or use the values file in step 1 using `--set-values`